### PR TITLE
Add disabled user store option

### PR DESCRIPTION
**Pull request by PasinduYeshan**
image

### Related Issues
- https://github.com/wso2/product-is/issues/16980

### CODE REVIEW

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 1]
```diff
@@ -877,38 +872,55 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
...
-                                                        disabled={ false }
-                                                        value={ selectedUserStore }
-                                                        onChange={
-                                                            (e: React.ChangeEvent<HTMLInputElement>,
-                                                                data: DropdownProps) => {
-                                                                setSelectedUserStore(data.value.toString());
-                                                                setSelectedUserStore(data.value.toString());
+                                        <>
+                                            <Grid.Row columns={ 1 } className="mb-0 pb-0">
+
```
Comment (dasuni-30) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:877]: Let's remove the redundant line?

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 2]
```diff
@@ -877,38 +872,55 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
...
+                                                    </Alert>
+                                                
+                                                </Grid.Column>
+                                            </Grid.Row>
+                                            <Grid.Row columns={ 1 }>
+                                                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 10 }> 
+                                                    <Form.Field required={ true }>
+                                                        <label className="pb-2">
+                                                            { t("console:manage.features.user.forms."+
+                                                                            "addUserForm.inputs.domain.placeholder") }
```
Comment (dasuni-30) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:892]: Let's fix indentation?

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 3]
```diff
@@ -877,38 +872,55 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
...
-                                                                setSelectedUserStore(data.value.toString());
-                                                                setSelectedUserStore(data.value.toString());
+                                        <>
+                                            <Grid.Row columns={ 1 } className="mb-0 pb-0">
+                                            
+                                                <Grid.Column mobile={ 16 } tablet={ 16 } computer={ 16 }> 
+                                                    <Alert severity="info">
+                                                        { t("console:manage.features.user.modals." +
+                                                        "bulkImportUserWizard.wizardSummary." +
+                                                    "disabledSecondaryStoreInfo") }
```
Comment (dasuni-30) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:882]: Let's fix indentation?

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 4]
```diff
@@ -877,38 +872,55 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
...
-                                                        data-componentid={ `${componentId}-userstore-dropdown` }
-                                                        name="userstore"
-                                                        disabled={ false }
-                                                        value={ selectedUserStore }
-                                                        onChange={
-                                                            (e: React.ChangeEvent<HTMLInputElement>,
-                                                                data: DropdownProps) => {
-                                                                setSelectedUserStore(data.value.toString());
-                                                                setSelectedUserStore(data.value.toString());
+                                        <>
```
Comment (Achintha444) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:875]: Remove this unnecessary component.

File: modules/i18n/src/translations/si-LK/portals/console.ts [hunk 5]
```diff
@@ -8784,7 +8784,8 @@ export const console: ConsoleNS = {
                             },
                             advanceSearch: {
                                 placeholder: "පරිශීලක නාමයෙන් සොයන්න"
-                            }
+                            },
+                            disabledSecondaryStoreInfo: "බාහිර පරිශීලක වෙළඳසැල් වෙත තොග ආනයනය මේ මොහොතේ ලබා ගත නොහැක."
```
Comment (Achintha444) [inline modules/i18n/src/translations/si-LK/portals/console.ts:8788]: Shall we rephrase this ?

**Comment by Achintha444** [comment]
Add changeset

File: modules/i18n/src/translations/si-LK/portals/console.ts (hunk 5 above)
Comment (PasinduYeshan) [reply modules/i18n/src/translations/si-LK/portals/console.ts]: I rephrased this as follows. WDYT?
"බාහිර පරිශීලක වෙළඳසැල් වෙත පරිශීලක තොග ආනයනය මේ මොහොතේ ලබා දිය නොහැක."

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx (hunk 4 above)
Comment (PasinduYeshan) [reply apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:875]: Since there are two `Grid.Rows` this component is required.

**Comment by PasinduYeshan** [comment]
This PR addresses claim dialect not resolving  in sub org issue as well.

File: apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx [hunk 6]
```diff
@@ -15,6 +15,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import Alert from "@oxygen-ui/react/Alert";
```
Comment (dasuni-30) [inline apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx:18]: Shall we have a new line

**Review by dasuni-30 — APPROVED** [review]

**Review by dasuni-30 — APPROVED** [review]

**Pull request merged**